src/headless/infra/variables.tf

```diff
@@ -10,10 +10,10 @@ variable "environment" {
   default     = "dev"
 }
 
-variable "locustVersion" {
-  description = "Locust Container Image Version"
+variable "locust_container_image" {
+  description = "Locust Container Image"
   type        = string
-  default     = "locustio/locust:1.4.3"
+  default     = "sebader/locust-with-plugins:latest"
```

Comment:
```suggestion
  default     = "sebader/locust-with-plugins:latest" # customized locust image with application insights support instead of locustio/locust via https://github.com/sebader/locust-with-plugins
```